### ISSUE TITLE
PSD not working anymore - user NOT in sudoers

### ISSUE THREAD

**Issue by dsiggi** [graysky2/profile-sync-daemon#373]
Hello,
psd ist not working for me anymore.

Here is the output from "journalctl":
```
Feb 25 09:47:32 nitro systemd[7033]: Starting Profile-sync-daemon...
Feb 25 09:47:32 nitro systemd[7033]: Finished Profile-sync-daemon.
Feb 25 09:47:32 nitro systemd[7033]: Reached target Main User Target.
Feb 25 09:47:32 nitro systemd[7033]: Starting Timed resync...
Feb 25 09:47:32 nitro sudo[7058]:    siggi : PWD=/home/siggi ; USER=root ; COMMAND=/usr/bin/psd-overlay-helper
Feb 25 09:47:33 nitro sudo[7220]:    siggi : PWD=/home/siggi ; USER=root ; COMMAND=/usr/bin/psd-overlay-helper -v 23 -l /home/siggi/.config/google-chrome-backup -u /run/user/1000/siggi-google-chrome-rw -w /run/us>
Feb 25 09:47:33 nitro sudo[7220]: pam_unix(sudo:session): session opened for user root(uid=0) by siggi(uid=1000)
Feb 25 09:47:33 nitro sudo[7224]:     root : user NOT in sudoers ; PWD=/home/siggi ; USER=siggi ; COMMAND=/usr/sbin/test -w /home/siggi/.config/google-chrome-backup
Feb 25 09:47:33 nitro profile-sync-daemon[7224]: »root« ist nicht in der sudoers-Datei.
Feb 25 09:47:33 nitro profile-sync-daemon[7224]: Dieser Vorfall wurde an den Administrator gemeldet.
Feb 25 09:47:33 nitro profile-sync-daemon[7222]: User siggi has no write permissions for /home/siggi/.config/google-chrome-backup. Aborting...
Feb 25 09:47:33 nitro sudo[7220]: pam_unix(sudo:session): session closed for user root
Feb 25 09:47:33 nitro profile-sync-daemon[7048]: Error in trying to mount /run/user/1000/siggi-google-chrome - this should not happen!
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Main process exited, code=exited, status=1/FAILURE
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Failed to kill control group /user.slice/user-1000.slice/user@1000.service/app.slice/psd-resync.service, ignoring: Operation not permitted
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Killing process 7226 (sudo) with signal SIGKILL.
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Killing process 7227 (sendmail) with signal SIGKILL.
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Failed to kill control group /user.slice/user-1000.slice/user@1000.service/app.slice/psd-resync.service, ignoring: Operation not permitted
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Failed to kill control group /user.slice/user-1000.slice/user@1000.service/app.slice/psd-resync.service, ignoring: Operation not permitted
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Killing process 7226 (sudo) with signal SIGKILL.
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Killing process 7227 (sendmail) with signal SIGKILL.
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Failed to kill control group /user.slice/user-1000.slice/user@1000.service/app.slice/psd-resync.service, ignoring: Operation not permitted
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Failed with result 'exit-code'.
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Unit process 7226 (sudo) remains running after unit stopped.
Feb 25 09:47:33 nitro systemd[7033]: psd-resync.service: Unit process 7227 (sendmail) remains running after unit stopped.
Feb 25 09:47:33 nitro systemd[7033]: Failed to start Timed resync.
```

Here is my sudoers file:
```
...
siggi ALL=(ALL) NOPASSWD: /usr/bin/psd-overlay-helper
%sudo ALL=(ALL:ALL) ALL
...
```

It seems that psd /home/siggi/.config/google-chrome-backup, but that's not true. ``touch /home/siggi/.config/google-chrome-backup/testfile`` will work.

I hope someone can help me.

**Comment by graysky2** [comment]
What is the output of:
```
mount|grep siggi
```

**Comment by dsiggi** [comment]
Hi,
```
mount | grep siggi
cryfs@/home/siggi/Dokumente/HomeBank on /home/siggi/HomeBank type fuse.cryfs (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
```

I also saw that whenn I start psd ```systemctl --user start psd``` the directories google-chrome will be moved do google-chrome-backup and google-chrome is than a symlink to /dev/null

**Comment by dsiggi** [comment]
I have been using psd and cryfs side by side for years without any problems.

For testing I deactivated the mount.
```
$ umount Homebank
$ systemctl --user start psd
$ mount | grep siggi
```

No output from ``mount | grep siggi``.
The output from ``journalctl`` is the same as above.

**Comment by dsiggi** [comment]
Afer a long time withoud using psd I find the solution of my problem.

On gentoo linux sudo is not installed by default. I installed sudo specifically for psd but gentoo does not create a default configuration file. So I just added the line "siggi ALL=(ALL) NOPASSWD: /usr/bin/psd-overlay-helper" with visudo.

But that is not enough. The line "root ALL=(ALL) ALL" was missing. With this line everything works again as usual.
